pkg/operator/ceph/cluster/cluster.go

```diff
@@ -290,48 +295,31 @@ func clusterChanged(oldCluster, newCluster cephv1.ClusterSpec, clusterRef *clust
 	return false, ""
 }
 
-func (c *cluster) unsetOrchestrationRunning() {
-	c.orchRunMux.Lock()
-	c.orchestrationRunning = false
-	c.orchRunMux.Unlock()
-}
-
-func (c *cluster) setOrchestrationPending() {
-	c.orchPenMux.Lock()
-	c.orchestrationPending = true
-	c.orchPenMux.Unlock()
-}
-
-func (c *cluster) getOrchestrationRunning() bool {
-	c.orchRunMux.Lock()
-	defer c.orchRunMux.Unlock()
-	return c.orchestrationRunning
+func (c *cluster) setOrchestrationNeeded() {
+	c.orchMux.Lock()
+	c.orchestrationNeeded = true
+	c.orchMux.Unlock()
 }
 
-func (c *cluster) getOrchestrationPending() bool {
-	c.orchPenMux.Lock()
-	defer c.orchPenMux.Unlock()
-	return c.orchestrationPending
+// unsetOrchestrationStatus resets the orchestrationRunning-flag
+func (c *cluster) unsetOrchestrationStatus(err error) {
```

Comment:
`err` is an unused parameter here.

Reply:
yeah i'd suggest we remove the err parameter since the err is logged elsewhere now